index.html

```diff
@@ -1,48 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Awesome Books</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Awesome Books</title>
 </head>
 <body>
-  <h1>Awesome books</h1> <br> <br>
-  <table class="table table-striped">
-    <thead>
-      <tr class="t-raw">
-        <th>Title</th>
-        <th>Author</th>
-        <th>ID</th>
-        <th></th>
-      </tr>
-    </thead>
-    <tbody id="book-list" class="bookList"></tbody>
-  </table> <br> <br>
-  <hr>
-  <br>
-  <form action="" id="book-form">
-    <div>
-      <label for="title"></label>
-      <input type="text" id="title" placeholder="Tittle" class="form-control">
+  <nav class="nav-bar navbar border">
+    <span>Awesome Books</span>
+    <ul class="ulNav">
+      <li class="border-right">
+        <a onclick="show(s1)">List |</a>
+      </li>
+      <li class="border-right">
+        <a onclick="show(s2)">Add Book |</a>
+      </li>
+      <li>
+        <a onclick="show(s3)">Contact</a>
+      </li>
+    </ul>
+  </nav>
+  <div class="timeDiv">
+    <span id="currentTime"></span>
+  </div>
+  <section class="books-list visible" id="s1">
+    <h1>Awesome books</h1>
+    <table class="table table-striped">
+      <thead>
+        <tr class="t-raw">
+          <th>Title</th>
+          <th>Author</th>
+          <th>ID</th>
+          <th></th>
+        </tr>
```

Comment:
- [ ] Either the list data should come below the corresponding list heading or we should delete the heading for the list. It will help us to make the design more accurate. Kindly try to fix this issue.

![image](https://user-images.githubusercontent.com/53019916/130230538-f5be731a-d118-4aaf-9fc4-035f1e951cd4.png)

Reply:
Resolved